### ISSUE TITLE
Slicing issue

### ISSUE THREAD

**Issue by mabelisle** [netaddr/netaddr#214]
Slicing is not showing the last IP in this situation:
I'm trying to show the first usable IP if the subnet is split in /27)

`ip = IPNetwork('10.96.0.0/24')`
`ip_list = list(ip[1::32])`

result
`ip_list = [IPAddress('10.96.0.1'), IPAddress('10.96.0.33'), IPAddress('10.96.0.65'), IPAddress('10.96.0.97'), IPAddress('10.96.0.129'), IPAddress('10.96.0.161'), IPAddress('10.96.0.193')]`

expected
`ip_list = [IPAddress('10.96.0.1'), IPAddress('10.96.0.33'), IPAddress('10.96.0.65'), IPAddress('10.96.0.97'), IPAddress('10.96.0.129'), IPAddress('10.96.0.161'), IPAddress('10.96.0.193'), IPAddress('10.96.0.225')]`

**Comment by metolone** [comment]
I saw the same issue using 0.8.0.  Note the difference slicing the object first vs. slicing the entire list:
```
import netaddr
network = netaddr.IPNetwork('172.16.0.0/22')
print(list(network[1::256]))   # list the slice
print(list(network)[1::256])   # slice the list
```
Output:
```
[IPAddress('172.16.0.1'), IPAddress('172.16.1.1'), IPAddress('172.16.2.1')]
[IPAddress('172.16.0.1'), IPAddress('172.16.1.1'), IPAddress('172.16.2.1'), IPAddress('172.16.3.1')]
```

**Comment by jstasiak** [comment]
Fix released in 1.1.0.
